py3status/modules/battery_level.py

```diff
@@ -240,11 +239,7 @@ def _parse_battery_info(acpi_battery_lines):
 
             return battery
 
-        acpi_raw = subprocess.check_output(
-            ["acpi", "-b", "-i"],
-            stderr=subprocess.STDOUT)
-
-        acpi_list = acpi_raw.decode("UTF-8").split('\n')
+        acpi_list = self.py3.command_output(["acpi", "-b", "-i"]).split('\n')
```

Comment:
`splitlines()` it makes little difference here because all line endings are transformed but split('\n') will cause issues one day so good to never use it.